hokusai/production.yml

```diff
@@ -200,11 +258,14 @@ spec:
               - sh
               - -c
               - sleep 5
+        command: ["/usr/local/bin/docker-entrypoint.sh", "/usr/local/bin/load_secrets_and_run.sh"]
+        args: ["/bin/sh", "-c", "./server.sh"]
       dnsPolicy: ClusterFirst
       dnsConfig:
         options:
         - name: ndots
           value: '1'
+      serviceAccountName: artsy-unleash-proxy
```

Comment:
i planned to just use [one service account](https://github.com/artsy/substance/blob/98606010adefa882a227f986cae29d3b5dd13263/services/service-accounts-k8s-apps.yml#L32) on k8s side. can just use `artsy-unleash` here.

Reply:
ouch, i am now seeing that there are two configmaps in this project. and they conflict in var values, e.g. `UNLEASH_URL`.

Reply:
that means Vault has to mirror that. we have to have two prefixes, one per deployment:

```
kvv2/data/kubernetes/apps/artsy-unleash/*
kvv2/data/kubernetes/apps/artsy-unleash-proxy/*
```

the first one already [exists](https://github.com/artsy/infrastructure/blob/8466051372ced68ba26ff00d8e835391efc7e95d/terraform/staging/vault/k8s_apps_role_policy.tf#L22). we have to add the 2nd one.

Reply:
so yeah, may as well add a `artsy-unleash-proxy` service account, too.

Reply:
Created service account -- https://github.com/artsy/substance/pull/402
Created aws role and policy -- https://github.com/artsy/infrastructure/pull/746
Created a path in vault 
```
vault kv put -mount kvv2 -cas=0 kubernetes/apps/artsy-unleash-proxy/PLACEHOLDER PLACEHOLDER=placeholder
```